docs/rum_collection/advanced_configuration.md

```diff
@@ -290,11 +449,33 @@ let session = URLSession(
     delegateQueue: nil
 )
 ```
+{{% /tab %}}
+{{% tab "Objective-C" %}}
+```objective-c
+DDConfigurationBuilder *builder = [DDConfiguration builderWithRumApplicationID:@"<rum_application_id>"
+                                                                   clientToken:@"<client_token>"
+                                                                   environment:@"<environment_name>"];
+
+// ...
+[builder trackUIKitRUMViews];
+[builder trackURLSessionWithFirstPartyHosts:[NSSet setWithObjects:@"example.com", nil]];
+
+DDGlobal.rum = [[DDRUMMonitor alloc] init];
+DDGlobal.sharedTracer = [[DDTracer alloc] initWithConfiguration:[DDTracerConfiguration new]];
+
+[DDDatadog initializeWithAppContext:[DDAppContext new]
+                    trackingConsent:trackingConsent
+                        configuration:[builder build]];
+```
+{{% /tab %}}
+{{< /tabs >}}
+
 This tracks all requests sent with the instrumented `session`. Requests matching the `example.com` domain are marked as "first party" and tracing information is sent to your backend to [connect the RUM resource with its Trace][10].
 
 To add custom attributes to resources, use the `.setRUMResourceAttributesProvider(_ :)` option when configuring SDK. By setting attributes provider closure, you can return additional attributes to be attached to tracked resource. 
 
 For instance, you may want to add HTTP request and response headers to the RUM resource:
+
 ```swift
 .setRUMResourceAttributesProvider { request, response, data, error in
```

Comment:
This API is not available in Objective-C, I believe it's a miss. WDYT @ncreated @bmermet ?

Reply:
Yes, it seems like we forgot to expose it. Let's add this to backlog 👌.

Reply:
RUMM-1914